README.md

```diff
@@ -245,6 +245,8 @@ Setting an ODJ Request file path for a configuration that creates a computer acc
 ### Unreleased
 
 * xAdDomainController: Update to complete fix for SiteName being required field.
+* xADDomain: Added retry logic to prevent FaultException to crash in Get-TargetResource on subsequent reboots after a domain is created because the service is not yet running. This error is mostly occur when the resource is used with the DSCExtension on Azure. 
```

Comment:
Since multiple contributors in #73 have confirmed that this fix works, I will skip adding tests for now and open an issue to add tests for this fix later.